docs/topics/ksp/ksp-faq.md

```diff
@@ -1,19 +1,24 @@
-[//]: # (title: Frequently asked questions)
+[//]: # (title: FAQ)
 
-## Why KSP?
+### Why KSP?
```

Comment:
Why thrid level headers?

Reply:
[Kotlin FAQ](https://kotlinlang.org/docs/faq.html) uses third-level headers. So this is for consistency.